src/__tests__/createAction-test.js

```diff
@@ -93,11 +93,6 @@ describe('createAction()', () => {
         type
       });
 
-      const explictNullAction = createAction(type)(null);
-      expect(explictNullAction).to.deep.equal({
-        type
-      });
-
```

Comment:
Was this failing before?

Also, I'm wondering if there's a test somewhere that says the following:

```js
  const explictNullAction = createAction(type)(null);
  expect(explictNullAction).to.deep.equal({
  type,
  payload: null,
});